frontend/src/components/dialogs/BlueprintSelectorDialog.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-    <ff-dialog ref="dialog" data-el="blueprint-selector-dialog" class="blueprints-selector-dialog" header="Choose a Blueprint">
+    <ff-dialog ref="dialog" data-el="blueprint-selector-dialog" class="blueprints-selector-dialog" header="Choose a Blueprint" style="display: block;">
```

Comment:
??

Reply:
It's best explained here: https://github.com/FlowFuse/flowfuse/issues/4028

Reply:
Right, but that's not in scope of this task, and doesn't explain why you've got hardcoded in-line `style` tag here?

Reply:
The hardcoded inline `style` tag was just laziness on my behalf, I'll address it shortly.

#4028 contains the a more generic explanation of why I propose this approach when it comes to dialogs, but for this case in particular the setTimeout on the flowRenderer was causing problems on the instance form because although the dialog was hidden, it was still present in the DOM and was reacting to state changes.

In this particular case, It was trying to render flows even when it didn't have a DOM element to attach itself to right after creating an instance before the page would have been redirected to the instance details page.

Reply:
I removed the inline style attribute